app/models/journeyDomain/consignment/ConsignmentDomain.scala

```diff
@@ -54,11 +52,5 @@ object ConsignmentDomain {
     }
   }
 
-  private def consigneeReader(implicit phaseConfig: PhaseConfig): Read[Option[ConsignmentConsigneeDomain]] =
-    phaseConfig.phase match {
-      case Phase.Transition =>
-        MoreThanOneConsigneePage.filterOptionalDependent(!_)(ConsignmentConsigneeDomain.userAnswersReader)
-      case Phase.PostTransition =>
-        ConsignmentConsigneeDomain.userAnswersReader.toOption
-    }
+  private def consigneeReader: Read[Option[ConsignmentConsigneeDomain]] = ConsignmentConsigneeDomain.userAnswersReader.toOption
```

Comment:
Is `MoreThanOneConsigneePage` now obsolete? Can the controller/page/view/message keys be removed?